Makefile

```diff
@@ -0,0 +1,42 @@
+.DEFAULT_GOAL := help
+
+PHPNOGC=php -d zend.enable_gc=0
+
+.PHONY: help
+help:
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+
+##
+## Commands
+##---------------------------------------------------------------------------
+
+clean: 	 ## Clean all created artifacts
+.PHONY: clean
+clean:
+	git clean --exclude=.idea/ -ffdx
```

Comment:
this is a super scary clean script 😬

Reply:
Just don't do it when you have uncommitted changes :P If you have a way to check that so we could bail out at that time that would be 👌 I didn't find anything :/

Note that it has never been a big deal for me because even when I did the mistake PHPStorm has always been able to restore everything without an issue